tests/cpp-tests/Classes/FileUtilsTest/FileUtilsTest.cpp

```diff
@@ -795,6 +795,8 @@ void TestWriteValueMap::onEnter()
     ValueMap mapInValueMap;
     mapInValueMap["string1"] = "string in dictInMap key 0";
     mapInValueMap["string2"] = "string in dictInMap key 1";
+    mapInValueMap["none"].getType();
```

Comment:
> What does this code do? Create a none element? If so then it is strange to do it like this.

Yes it create none element. I want to say that anyone create none element if they intended or not.